specification/app/resource-manager/Microsoft.App/preview/2024-02-02-preview/ContainerApps.json

```diff
@@ -695,6 +698,14 @@
           ],
           "description": "The fully qualified resource ID of the resource that manages this resource. Indicates if this resource is managed by another Azure resource. If this is present, complete mode deployment will not delete the resource if it is removed from the template since it is managed by another resource."
         },
+        "kind": {
+          "type": "string",
```

Comment:
Model it as an enums because the property has a finite set of values.
refs: https://armwiki.azurewebsites.net/api_contracts/guidelines/openapi.html#oapi015-avoid-modelling-a-property-as-string-type-when-it-should-be-an-enums

Reply:
But for this API, there're limited allowed values for the `kind` field, is it correct? If so, model it as enum will help users to use this API.